robot-command/src/main/java/org/obolibrary/robot/TemplateCommand.java

```diff
@@ -137,7 +135,16 @@ public CommandState execute(CommandState state, String[] args) throws Exception
       tables.put(templatePath, TemplateHelper.readTable(templatePath));
     }
 
-    OWLOntology outputOntology = TemplateOperation.template(tables, inputOntology, null, ioHelper);
```

Comment:
Switching to a `Template` class makes sense and is fine, but I want to maintain our overall design pattern: `TemplateCommand` should call a static method on `TemplateOperation`.

So I would prefer to define a `TemplateOperation.template2(...)` method, even if all it does is run this little bit of code that instantiates a new `Template`.